src/Type/WebMozartAssert/AssertTypeSpecifyingExtension.php

```diff
@@ -416,7 +416,40 @@ private function getExpressionResolvers(): array
 								new Name($classNames[0])
 							);
 						}
-						return null;
+
+						if ($classType->isClassStringType()->yes()) {
+							$templateClasses = $classType->getReferencedClasses();
```

Comment:
This isn't the right method. If you have for example `class-string<Foo<Bar>>`, it's going to return both Foo and Bar.

I think the right method here is `$classType->getClassStringObjectType()->getObjectClassNames()`.

Also consider if `getObjectTypeOrClassStringObjectType` isn't the right method instead.

Reply:
Didn't think of that! Will add an extra test to cover it